en/docs/install-and-setup/install-and-setup-overview.md

```diff
@@ -0,0 +1,23 @@
+# Install and Setup Overview
```

Comment:
This page looks incomplete. Shall we leave this out?

Reply:
Ok